client/objects.lua

```diff
@@ -43,136 +43,49 @@ function GetClosestSpike()
-        flags = 16,
-    }, {}, {}, function() -- Done
+RegisterNetEvent('police:client:spawnPObj', function(item)
+    if lib.progressBar({
+        duration = 2500,
+        label = Lang:t("progressbar.place_object"),
+        useWhileDead = false,
+        canCancel = true,
+        disable = {car = true, move = true, combat = true},
+        anim = {
+            dict = "anim@narcotics@trash",
+            clip = "drop_front"
+        }
+    }) then
         StopAnimTask(PlayerPedId(), "anim@narcotics@trash", "drop_front", 1.0)
-        TriggerServerEvent("police:server:spawnObject", "tent")
-    end, function() -- Cancel
-        StopAnimTask(PlayerPedId(), "anim@narcotics@trash", "drop_front", 1.0)
-        QBCore.Functions.Notify(Lang:t("error.canceled"), "error")
-    end)
-end)
-
-RegisterNetEvent('police:client:spawnLight', function()
-    QBCore.Functions.Progressbar("spawn_object", Lang:t("progressbar.place_object"), 2500, false, true, {
-        disableMovement = true,
-        disableCarMovement = true,
-        disableMouse = false,
-        disableCombat = true,
-    }, {
-        animDict = "anim@narcotics@trash",
-        anim = "drop_front",
-        flags = 16,
-    }, {}, {}, function() -- Done
-        StopAnimTask(PlayerPedId(), "anim@narcotics@trash", "drop_front", 1.0)
-        TriggerServerEvent("police:server:spawnObject", "light")
-    end, function() -- Cancel
+        TriggerServerEvent("police:server:spawnObject", item)
+    else
+        lib.cancel()
         StopAnimTask(PlayerPedId(), "anim@narcotics@trash", "drop_front", 1.0)
-        QBCore.Functions.Notify(Lang:t("error.canceled"), "error")
-    end)
+        lib.notify({description = Lang:t("error.canceled"), type = "error"})
+    end
 end)
 
 RegisterNetEvent('police:client:deleteObject', function()
     local objectId, dist = GetClosestPoliceObject()
     if dist < 5.0 then
-        QBCore.Functions.Progressbar("remove_object", Lang:t('progressbar.remove_object'), 2500, false, true, {
-            disableMovement = true,
-            disableCarMovement = true,
-            disableMouse = false,
-            disableCombat = true,
-        }, {
-            animDict = "weapons@first_person@aim_rng@generic@projectile@thermal_charge@",
-            anim = "plant_floor",
-            flags = 16,
-        }, {}, {}, function() -- Done
+        if lib.progressBar({
+            duration = 2500,
+            label = Lang:t("progressbar.remove_object"),
+            useWhileDead = false,
+            canCancel = true,
+            disable = {car = true, move = true, combat = true},
+            anim = {
+                dict = "weapons@first_person@aim_rng@generic@projectile@thermal_charge@",
+                clip = "plant_floor",
+            }
+        }) then
             StopAnimTask(PlayerPedId(), "weapons@first_person@aim_rng@generic@projectile@thermal_charge@", "plant_floor", 1.0)
             TriggerServerEvent("police:server:deleteObject", objectId)
```

Comment:
```suggestion
            TriggerServerEvent("police:server:deleteObject", objectId)
```
Line can be completely removed, ox_lib already stops the animation.